source/Halibut.Tests/DataStreamFixture.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Halibut.Tests.Support;
+using Halibut.Tests.Support.TestAttributes;
+using Halibut.Tests.Support.TestCases;
+using Halibut.Tests.TestServices.Async;
+using Halibut.TestUtils.Contracts;
+using Halibut.Transport.Streams;
+using NUnit.Framework;
+
+namespace Halibut.Tests
+{
+    public class DataStreamFixture : BaseTest
+    {
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false)]
+        [LatestClientAndPreviousServiceVersionsTestCases(testNetworkConditions: false)]
+        [Obsolete]
+        public async Task ASyncDataStreamWriterCanStillBeUsed(ClientAndServiceTestCase clientAndServiceTestCase)
```

Comment:
Is this supposed to be `SyncDataStreamWriterCanStillBeUsed` instead?

I wondered why an async test was marked as `Obsolete` straight away.

Reply:
No no its "A" "Sync" as to distinct words. Do you even camel case? :p 

I will change it